jupyter-lib/api/src/main/kotlin/org/jetbrains/kotlinx/jupyter/api/TypeRenderersProcessor.kt

```diff
@@ -1,13 +1,12 @@
-package org.jetbrains.kotlinx.jupyter.codegen
+package org.jetbrains.kotlinx.jupyter.api
 
-import org.jetbrains.kotlinx.jupyter.api.Code
-import org.jetbrains.kotlinx.jupyter.api.FieldValue
-import org.jetbrains.kotlinx.jupyter.api.RendererTypeHandler
 import org.jetbrains.kotlinx.jupyter.api.libraries.ExecutionHost
 
 interface TypeRenderersProcessor {
 
     fun renderResult(host: ExecutionHost, field: FieldValue): Any?
 
+    fun renderValue(host: ExecutionHost, value: Any?): Any?
```

Comment:
It is not clear what this function does. Documentation would be nice.

Reply:
I see a problem here. Suppose the value is a container (`List<Something>`). The type of actual element renderer could not be resolved in the runtime due to type erasure. Won't it require KType?

Reply:
Regarding documentation: I've added it.
Regarding renderers: yes, due to type erasure KClass is the most we can get in runtime. It is how all the rendering mechanics works, it only uses KClass for getting type information. We could add Ktype argument here, but it would require breaking the renderers API, and it will actually be useless because it cannot be used for cells results rendering.